tensorflow/core/kernels/mkl_pooling_ops_common.cc

```diff
@@ -228,7 +226,7 @@ void MklPoolingBwdPrimitive<T>::Setup(const MklPoolingParams& bwdParams) {
     context_.bwd.reset(
         new pooling_backward(*context_.bwd_pd, *context_.diff_dst_mem,
                              *context_.ws_mem, *context_.diff_src_mem));
-#endif //ENABLE_MKLDNN_V1
+#endif // ENABLE_MKLDNN_V1
```

Comment:
Unfortunately this still isn't fixed yet. In my last review, I was trying to say that there should be two spaces between `#endif` and `//`. Since you'll have to make one more edit anyway, would you mind also fixing this? Thank you!
```suggestion
#endif  // ENABLE_MKLDNN_V1
```

Reply:
and I almost made all others to single space including
```#endif  // INTEL_MKL
#endif  // TENSORFLOW_CORE_UTIL_MKL_UTIL_H_```

Luckily I did not. Fixed with two spaces.